test/detached.js

```diff
@@ -27,7 +27,7 @@ describe('detached DOM node', function() {
       if (typeof MutationObserver === 'function') {
         describe('when the browser supports `MutationObserver`', function () {
 
-          beforeEach(h.wait(50));
```

Comment:
Not really relevant, since a `setTimeout()` is set every 150ms :cry: 
The `MutationObserver` test seems to always fail when there is no `setTimeout()`.